pan_firewall/README.md

```diff
@@ -60,13 +60,13 @@ Firewall authentication logs provide detailed information on users as they authe
 
 ## Data Collected
 
-### Logs
+### Metrics
 
-The PANOS integration collects logs from the Palo Alto Networks firewall integration and forwards them to Datadog.
+The Palo Alto Networks Firewall Log Integration submits specified metrics under the `snmp.*` namespace. The metrics collected are determined by the [palo alto configured profile][9]. See the [SNMP Data Collected][10] section for details of monitored metrics.
```

Comment:
Prevent merging

Reply:
Ok that makes sense, this was a request from Clint Kennedy , I'll check to see what they wanted here, thanks :) 
cc @angelinajin

Reply:
Discussed via slack/ contacting PM for next steps.